sktime/forecasting/base/_base.py

```diff
@@ -170,6 +170,27 @@ def __rmul__(self, other):
         else:
             return NotImplemented
 
+    def __or__(self, other):
+        """Magic | method, return MultiplexForecaster.
+
+        Implemented for `other` being either a MultiplexForecaster or a forecaster.
+
+        Parameters
+        ----------
+        other: `sktime` forecaster or sktime MultiplexForecaster
+
+        Returns
+        -------
+        MultiplexForecaster object
+        """
+        from sktime.forecasting.compose import MultiplexForecaster
+
+        # generate a name for our forecaster based on class name:
+        self_name = type(self).__name__.lower()
```

Comment:
I think the `MultiplexForecaster` shouhld take care of generating the name - I would just do `MultiplexForecaster(self)`

Reply:
ok!